.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "vunit"]
```

Comment:
Instead of adding VUnit as a submodule, I would suggest adding a `requirements.txt` file. pip supports installing any branch/commit from a GitHub repo, without pushing it to the PyPI registry. See, for instance: https://github.com/chipsalliance/f4pga-examples/blob/main/common/requirements.txt#L18 or https://github.com/ghdl/ghdl/blob/master/pyGHDL/dom/requirements.txt#L4.

By doing so, colorama would be installed as a dependency of VUnit, so it would not be necessary to install it explicitly.